### PR TITLE
PR to merge 'es6-module' to the 'main'

### PR DESCRIPTION
**Pull request by AliRezaBmeDu**
In this branch I have done the following changes.

- Create a directory called modules.
- Create the `bookDbConstruct` and `navigation` modules.
- Use the modules to create the awesome-books application.
- Add necessary designs.
- Use the ES6 syntax
- Add webpack
- Fix the linters for html-css-javascript

### CODE REVIEW

File: modules/navigation.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    case 'form':
+      formDiv.classList.remove('hide-seek');
+      break;
+    case 'contact':
+      contactDiv.classList.remove('hide-seek');
+      break;
+    default:
+      break;
+  }
+}
```
Comment (TedLivist) [inline modules/navigation.js]: - [ ] Kindly change this to an arrow function in keeping with the ES6 syntax requirement.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  listBtn.addEventListener('click', () => showSection('list'));
+  addBtn.addEventListener('click', () => showSection('form'));
+  contactBtn.addEventListener('click', () => showSection('contact'));
+
+  showSection('list');
+
+  function updateClock() {
+    const now = DateTime.now().toLocaleString(DateTime.DATETIME_FULL);
+    clockDiv.textContent = now;
+  }
```
Comment (TedLivist) [inline index.js]: - [ ] Kindly change this to an arrow function in keeping with the ES6 syntax requirement.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @AliRezaBmeDu,

Good job so far!
✔️ Use of arrow functions
✔️ Use of luxon for date and time
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**